### DISCUSSION TITLE
transfer pybamm model output to numpy arrays

### DISCUSSION THREAD

**Discussion by xyyrly** [pybamm-team/PyBaMM#3090] [Q&A] [answered]
Hi~
Question again:)
Are there some pybamm functions allowing to transfer model output to numpy arrays? I try to check the output but now I can only plot them using pybamm.dynamic_plot

Thank you guys!!!!

**Comment by brosaplanella** [comment, marked as answer]
This [notebook](https://docs.pybamm.org/en/latest/source/examples/notebooks/getting_started/tutorial-6-managing-simulation-outputs.html) explains how to handle the simulations output.

**Reply by xyyrly** [reply]
It helps a lot. Thank you Planella!
